### ISSUE TITLE
I am unable to install this plugin

### ISSUE THREAD

**Issue by aggarwalneeraj** [azachar/protractor-screenshoter-plugin#21]
Trying to install this plugin following instructions specified. Getting error. Error Log attached. Please guide.
[error_log.txt](https://github.com/azachar/protractor-screenshoter-plugin/files/788774/error_log.txt)

**Comment by azachar** [comment]
Dear @aggarwalneeraj,
it is related to your node installation. It has nothing to do with this plugin. Did you tried to install something else? 

Cheers,
Andrej

**Comment by aggarwalneeraj** [comment]
Hi Andrej,

Yes, you are correct. I faced issues with other installations as well. I did a clean NODE installation and then tried installing this plugin and it was done successfully.  Thanks for your response.

Now, when plugin is installed, I have 2 questions for you:
1. My test specs have few steps (like enter name, click here, and then click a checkbox and then submit button, all on the same page).  While capturing screenshots for a spec, I am seeing that screens were captured just midway, just after executing few steps. Test has passed though. 
2. While running same set of test cases (suite) on 2 different browsers (Chrome and Firefox say), report doesn't distinguishes between results/screenshots from different browsers. It just show spec 0-8 and then again spec 0-8. I am not sure if plugin has this feature or I am missing out on something?
3. For other test case with several steps and expects(), Screenshots are displayed in reverse order of its execution. Is there a way I can reverse their order so that it displays screenshots in order steps were listed?
4. From where can I get the list of various parameters (like screenshotPath, writeReportFreq etc.) of the plugin ?
5.  How do i default the report to open screenshot based report? and also I want to have default settings for Expand All, Show screenshots, Show Passed Expectations, Show Protractor Stack etc?
